content/en/dora_metrics/setup/deployments.md

```diff
@@ -0,0 +1,216 @@
+
+[1]: https://docs.datadoghq.com/integrations/github/
+[2]: https://app.datadoghq.com/integrations/github/
+{{% /tab %}}
+
+{{% tab "Other Git Providers" %}}
+
+You can upload your Git repository metadata with the [`datadog-ci git-metadata upload`][1] command.
+When this command is executed, Datadog receives the repository URL, the commit SHA of the current branch, and a list of tracked file paths.
+
+Run this command in CI for every new commit. If a deployment is executed for a specific commit SHA, ensure that the `datadog-ci git-metadata upload` command is run for that commit **before** the deployment event is sent.
+
+<div class="alert alert-warning">
+Do not provide the <code>--no-gitsync</code> option to the <code>datadog-ci git-metadata upload</code> command.
+When that option is included, the commit information is not sent to Datadog and the Change Lead Time metric is not calculated.
+</div>
+
+You can validate the correct setup of the command by checking the command output. An example of a correct output is:
+```
+Reporting commit 007f7f466e035b052415134600ea899693e7bb34 from repository git@github.com:organization/example-repository.git.
+180 tracked file paths will be reported.
+✅  Handled in 0.077 seconds.
+```
+
+[1]: https://github.com/DataDog/datadog-ci/tree/master/src/commands/git-metadata
+{{% /tab %}}
+{{< /tabs >}}
+
+
+### Handling multiple services in the same repository
+
+If the source code of multiple services is present in the same repository, further actions are needed to ensure that the Change Lead Time is calculated by taking into account only the commits affecting the specific service being deployed.
+To filter the commits measured to only the ones that affect the service, specify the source code glob file path patterns in the [service definition][5].
+
+If the service definition contains a **full** GitHub URL to the application folder, a single path pattern is automatically used.
+
+**Example (schema version v2.2):**
+
+```yaml
+links:
+  - name: shopist
+    type: repo
+    provider: github
+    url: https://github.com/organization/example-repository/tree/main/src/apps/shopist
+```
+
+DORA Metrics for the `shopist` service only consider the git commits that include changes within `src/apps/shopist/**`. You can configure more granular control of the filtering with `extensions[datadoghq.com/dora-metrics]`.
+
+**Example (schema version v2.2):**
+
+```yaml
+extensions:
+  datadoghq.com/dora-metrics:
+    source_patterns:
+      - src/apps/shopist/**
+      - src/libs/utils/**
+```
+
+DORA Metrics for the service `shopist` only consider the Git commits that include changes within `src/apps/shopist/**` or `src/libs/utils/**`.
+
+### Limitations
+
+- The retention of Git metadata is 1 month. Commits older than 1 month might not be taken into account when computing Change Lead Time.
```

Comment:
can we bullet this out for readability? 

- The retention of Git metadata is 1 month. Commits older than 1 month might not be taken into account when computing Change Lead Time.
- Change Lead Time is not available for the first deployment of a service that includes Git information.